File: LP/C/ex2.c
#include <stdio.h>
#include <string.h>

void main() {
    int num;
    int cont = 0;

    printf("Digite um número: ");
    scanf("%d", &num);

    for (int i = 1; i <= num; i++){
        if (num % i == 0){
            cont++;
        }
    }

    if (cont == 2){
        printf("O número %d é primo\n", num);
    } else {
        printf("O número %d não é primo\n", num);
    }
}

File: LP/GO/ex2.go
package main

import(
	"fmt"
)

func main(){
	var(
		num int
		prime = true	
	)

	fmt.Print("Digite um número: ")
	fmt.Scan(&num)

	if num < 2{
		prime = false
	} else {
		for i := 1; i <= num / 2; i++{
			if num % i == 2{
				prime = false
				break
			}
		}
	}

	if prime {
		fmt.Printf("O número %d é primo\n", num)
	} else {
		fmt.Printf("O número %d não é primo\n", num)
	}
}
